Spigot-Server-Patches/0542-Return-chat-component-with-empty-text-instead-of-thr.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: CDFN <codefun@protonmail.com>
+Date: Tue, 7 Jul 2020 17:53:23 +0200
+Subject: [PATCH] Return chat component with empty text instead of throwing
+ exception
+
+
+diff --git a/src/main/java/net/minecraft/server/Container.java b/src/main/java/net/minecraft/server/Container.java
+index 268bedd042b651514aa94b735876de9c40bae56f..95c7dfd33beb576fa6221a025b459c824b4cbcdb 100644
+--- a/src/main/java/net/minecraft/server/Container.java
++++ b/src/main/java/net/minecraft/server/Container.java
+@@ -45,7 +45,12 @@ public abstract class Container {
+     }
+     private IChatBaseComponent title;
+     public final IChatBaseComponent getTitle() {
+-        Preconditions.checkState(this.title != null, "Title not set");
++        // Paper start - return chat component with empty text instead of throwing error
++        // Preconditions.checkState(this.title != null, "Title not set");
++        if(this.title == null){
++            return IChatBaseComponent.ChatSerializer.jsonToComponent("\"text\":\"\"");
```

Comment:
Does this even work? Would it not require being a JSON object?

Reply:
Good point, thanks 👍